packages/lib/environment/auth.ts

```diff
@@ -0,0 +1,31 @@
+import { prisma } from "@formbricks/database";
+import { Session } from "next-auth";
+
+export const hasUserEnvironmentAccess = async (user: Session["user"], environmentId: string) => {
```

Comment:
can we add a cache to this? maybe no revalidationTags, just 5 minute revalidation period?

Reply:
done